modules/ROOT/pages/index.adoc

```diff
@@ -1,48 +1,95 @@
-= Introduction to data migration
-:page-tag: migration,zdm,zero-downtime,zdm-proxy, introduction
+= {company} migration tools
+:navtitle: Data migration
+:page-layout: landing
 
-Enterprises today want to reliably migrate mission-critical client applications and data across environments with little or no downtime during the migration.
+{company} has developed a set of thoroughly tested self-service tools that can help you migrate your data from any origin cluster based on {cass-reg} to any {cass-short}-compatible target with little or no downtime.
 
-{company} has developed a set of thoroughly-tested self-service tools that can help you migrate your data from any {cass-short} origin cluster to any {cass-short}-compatible target.
+[.[&>h2]:!hidden]
+== {empty}
 
-Compatible origin and target clusters include {cass-reg}, {dse}, {hcd}, and {astra-db}.
+[subs="macros,attributes"]
+++++
+<div class="flex rounded bg-level1 p-6 gap-4">
+  <div class="flex flex-col lg:basis-1/2 gap-2">
 
-When the migration is complete, the data is present in the new database, and you can update your client applications to connect exclusively to the new database.
-The old database becomes obsolete and can be removed.
+    <h2 class="discrete !text-h1 !m-0 !mb-4">Secure, reliable migrations</h2>
 
-Available migration tools include:
+    <p class="!mb-2">Bulk-load terabytes while shifting traffic with little or no downtime, securely and reliably.</p>
 
-* xref:ROOT:introduction.adoc[{product}] ({product-short}): Comprised of {product-proxy}, {product-proxy}, and{product-automation}, you can continue to run your current application and migrate data from the origin to the target database without any downtime.
-{product-proxy} helps to manage the activity in transition.
+    <div class="flex gap-2 !mt-4">
+      xref:ROOT:introduction.adoc[Get started with {product},role="btn btn-primary btn-solid"]
+      xref:ROOT:components.adoc[Compare tools,role="btn btn-neutral btn-outlined"]
+    </div>
 
-* xref:ROOT:cassandra-data-migrator.adoc[{cass-migrator}]: It can be used in conjunction with the {product-proxy} for a migration with zero downtime. It can also be used on its own for migrations with acceptable downtime.
+  </div>
+  <div class="hidden lg:block flex basis-1/2 relative">
+++++
 
-* xref:ROOT:dsbulk-migrator.adoc[{dsbulk-migrator}]: {dsbulk-migrator} is an extension of {dsbulk-loader}.
-In addition to loading and unloading CSV and JSON data, {dsbulk-migrator} can transfer data between databases.
-It can read data from a table from your origin database and write it to a table in your target database.
-It can be used as an alternative to {cass-migrator} ({cass-migrator-short}).
+svg::sideloader:astra-migration-toolkit.svg[role="absolute bottom-1/2 translate-y-1/2 right-0 w-full"]
```

Comment:
@eric-schneider Did I copy something wrong? It's like offset

![image](https://github.com/user-attachments/assets/766cc713-d672-4bcf-ae2a-173313fe3aca)

Reply:
Works on my machine 🤔 Scales properly in both Firefox and Chrome, though it gets a little comical as it approaches the mobile break point 😅 

![image](https://github.com/user-attachments/assets/ee6ef993-0ccd-474f-ad8b-ab4565f33b62)

![image](https://github.com/user-attachments/assets/7dacf3cb-9c30-4e06-874b-2091b7c1ab8c)

Reply:
MAybe just weirdness with my browser then lol